packages/jest-core/src/TestScheduler.ts

```diff
@@ -419,11 +403,11 @@ class TestScheduler {
    * to make dealing with them less painful.
    */
   private _getReporterProps(reporter: string | Config.ReporterConfig): {
-    path: string;
     options: Record<string, unknown>;
+    path: string;
   } {
     if (typeof reporter === 'string') {
-      return {options: this._options, path: reporter};
+      return {options: {}, path: reporter};
```

Comment:
This was `startRun()`? Perhaps for notify reporter?

Reply:
Hm.. Possibly this branch is never called, because reporters are already normalised:

https://github.com/facebook/jest/blob/62b4bd701db378b38d360094f55fbd9ba64ca937/packages/jest-config/src/normalize.ts#L415-L429

Reply:
let's. remove it, probably for legacy reasons